website/README.md

```diff
@@ -1,6 +1,6 @@
-# OneTable Website Source Code
+# XTable Website Source Code
 
-This repo hosts the source code of [OneTable](https://onetable.dev)
+This repo hosts the source code of [XTable](https://onetable.dev)
```

Comment:
Shouldn't the link be updated to point to the GitHub Repo?

Reply:
I think it's intended to point to the website. 

but can we fix GH pages on the xtable repo and point the link to xtable.apache.org (which is broken atm)

Reply:
ya good call, I updated